cfme/tests/infrastructure/test_providers.py

```diff
@@ -149,7 +149,13 @@ def test_provider_add_with_bad_credentials(provider):
         with error.expected('Cannot complete login due to an incorrect user name or password.'):
             provider.create(validate_credentials=True)
     elif isinstance(provider, RHEVMProvider):
-        with error.expected('401 Unauthorized'):
+        with error.expected(
+            version.pick(
```

Comment:
I'd rather recommend to pick the message before the with, this does not look so nice.